doc/source/configure.rst

```diff
@@ -113,6 +113,27 @@ start a new worker with the given *root temporary directory*.
               ├── plasma_store
               └── raylet  # this could be deleted by Ray's shutdown cleanup.
 
+Ports configurations
+--------------------
+Ray requires bi-directional communication among its nodes in a cluster. Each of node is supposed to open specific ports to receive incoming network requests.
+
+.. code-block:: shell
+
+  # Head Node
+  --port # Port of GCS (Redis).
+  --node-manager-port # Raylet port for node manager.
+  --object-manager-port # Raylet port for object manager.
+  --gcs-server-port # GCS Server port. GCS server is a stateless service that is in charge of communicating to GCS.
```

Comment:
Can you make the head node and worker node sections their own blocks and then use plaintext formatting as richard suggested?